src/Microsoft.ML.AutoML/Microsoft.ML.AutoML.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>7.3</LangVersion>
```

Comment:
The root `Directory.Build.props` file defaults this to `latest`, so you shouldn't need to set this property.

You can also remove the `RootNamespace` element below.

Reply:
Removed.